app/js/lib/actions.js

```diff
@@ -40,6 +33,13 @@ module.exports = {
 			type: 'SELECT_PAYMENT_TYPE',
 			payload: { paymentType: paymentType }
 		};
+	},
+
+	newValidateAmountAction: function ( validationResult ) {
+		return {
+			type: 'VALIDATE_AMOUNT',
+			payload: validationResult
```

Comment:
The payload actually _is_ an object - it's the JSON object returned by validation call ( `{ status: 'OK' }` or `{ status: 'ERR', message: 'Amount too low' }`). I'll add doc comments to all the action creators when this is merged.

In this case `validationResult` can also be the `Promise` returned by the jQuery call. The `redux-promise` middleware will handle it in then and create a new action with the validation result object returned by the promise.